windows/vhba_hot_add_remove/nvme_disk_hot_extend_spec13.yml

```diff
@@ -9,17 +9,25 @@
   hosts: localhost
   gather_facts: false
   tasks:
-    - block:
-        - include_tasks: ../setup/test_setup.yml
-        - include_tasks: ../../common/skip_test_case.yml
+    - name: "Test case block"
+      block:
+        - name: "Test setup"
+          include_tasks: ../setup/test_setup.yml
+
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
           vars:
-            skip_msg: "Skip this test case due to ESXi version '{{ esxi_version }}' is not >= 7.0.2."
+            skip_msg: >- 
+              Skip this test case due to VM hardware version '{{ vm_hardware_version_num }}' is not >= 19, which is 
```

Comment:
"not >=" can be replaced with "less than". And I think the message can be more specific. For example,
```
The NVMe 1.3 Spec is supported from ESXi 7.0.2 hardware version 19. Skip this test case due to VM's hardware version is '{{ vm_hardware_version_num }}'.
```

Reply:
Sure. It's beter. Thanks.